src/Makefile.am

```diff
@@ -1,5 +1,5 @@
 lib_LTLIBRARIES = libstorj.la
-libstorj_la_SOURCES = storj.c utils.c utils.h http.c http.h uploader.c uploader.h downloader.c downloader.h bip39.c bip39.h bip39_english.h crypto.c crypto.h rs.c rs.h
+libstorj_la_SOURCES = storj.c utils.c utils.h http.c http.h uploader.c uploader.h downloader.c downloader.h bip39.c bip39.h bip39_english.h crypto.c crypto.h rs.c rs.h storjapi_callback.c storjapi_callback.h
```

Comment:
Instead of `storjapi_callback.c` and `storjapi_callback.h` maybe `cli_callback.c` ?

Reply:
Changed the file names from Storjapi_callback.h/c to cli_callback.h/c